File: CTDL1/New folder/Common7/IDE/ItemTemplates/VisualBasic/Office SharePoint/1033/SharePoint14BDC/Entity1Service.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

''' <summary>
''' $loc_Entity1ServiceSummary1$
''' $loc_Entity1ServiceSummary2$
''' </summary>
Public Class Entity1Service
    ''' <summary>
    ''' $loc_Entity1ServiceReadItemSummary$
    ''' $loc_Entity1ServiceFunctionSummary$
    ''' </summary>
    ''' <param name="id"></param>
    ''' <returns>Entity1</returns>
    Public Shared Function ReadItem(ByVal id As String) As Entity1
        '$loc_Entity1ServiceFunctionTodoComment$
        Dim entity1 As New Entity1()
        entity1.Identifier1 = id
        entity1.Message = "Hello World"
        Return entity1
    End Function

    ''' <summary>
    ''' $loc_Entity1ServiceReadListSummary$
    ''' $loc_Entity1ServiceFunctionSummary$
    ''' </summary>
    ''' <returns>$loc_Entity1ServiceReadListReturns$</returns>
    Public Shared Function ReadList() As IEnumerable(Of Entity1)
        '$loc_Entity1ServiceFunctionTodoComment$
        Dim entity1 As New Entity1()
        entity1.Identifier1 = "0"
        entity1.Message = "Hello World"
        Dim myEntityList As Entity1() = {entity1}
        Return myEntityList
    End Function
End Class

File: CTDL1/New folder/Common7/IDE/ItemTemplatesCache/CSharp/Office SharePoint/1033/SharePoint16BDC/Entity1Service.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace $rootnamespace$.$subnamespace$
{
    /// <summary>
    /// $loc_Entity1ServiceSummary1$
    /// $loc_Entity1ServiceSummary2$
    /// </summary>
    public class Entity1Service
    {
        /// <summary>
        /// $loc_Entity1ServiceReadItemSummary$
        /// $loc_Entity1ServiceFunctionSummary$
        /// </summary>
        /// <param name="id"></param>
        /// <returns>Entity1</returns>
        public static Entity1 ReadItem(string id)
        {
            //$loc_Entity1ServiceFunctionTodoComment$
            Entity1 entity1 = new Entity1();
            entity1.Identifier1 = id;
            entity1.Message = "Hello World";
            return entity1;
        }
        /// <summary>
        /// $loc_Entity1ServiceReadListSummary$
        /// $loc_Entity1ServiceFunctionSummary$
        /// </summary>
        /// <returns>$loc_Entity1ServiceReadListReturns$</returns>
        public static IEnumerable<Entity1> ReadList()
        {
            //$loc_Entity1ServiceFunctionTodoComment$
            Entity1[] entityList = new Entity1[1];
            Entity1 entity1 = new Entity1();
            entity1.Identifier1 = "0";
            entity1.Message = "Hello World";
            entityList[0] = entity1;
            return entityList;
        }
    }
}
